addons/binding/org.openhab.binding.mpower/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,31 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: mPower Binding
+Bundle-SymbolicName: org.openhab.binding.mpower;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/jsch-0.1.54.jar
+Import-Package: 
+ com.google.common.base;version="10.0.1",
+ com.google.common.collect;version="10.0.1",
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.jupnp.model.meta,
+ org.jupnp.model.types,
+ org.openhab.binding.mpower,
+ org.openhab.binding.mpower.handler,
+ org.osgi.framework,
+ org.osgi.service.component,
+ org.slf4j
+Service-Component: OSGI-INF/*.xml
+Export-Package: org.openhab.binding.mpower,
+ org.openhab.binding.mpower.handler
+Require-Bundle: org.apache.commons.lang
```

Comment:
According to the [Coding Guidelines](http://docs.openhab.org/developers/development/guidelines.html#b-osgi-bundles), the manifest must not contain any “Require-Bundle” entries. Instead, “Import-Package” must be used.